src/main/java/svc/models/Court.java

```diff
@@ -0,0 +1,156 @@
+package svc.models;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+// https://vladmihalcea.com/2017/03/29/the-best-way-to-map-a-onetomany-association-with-jpa-and-hibernate/
+
+@Entity
+public class Court {
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name="court_id")
+	private Long id;
+	
+	@Column(name="court_name")
+	private String name;
+	private String phone;
+	private String website;
+	private String extension;
+	private String address;
+	private String paymentSystem;
+	private String city;
+	private String state;
+	
+	@Column(name="zip_code")
+	private String zipCode;
+	private Double latitude;
+	private Double longitude;
+	
+	@OneToMany(cascade = CascadeType.ALL, orphanRemoval = true)
+	@JoinColumn(name = "court_id")
+	private List<Judge> judges;
+	
+	@Column(name="citation_expires_after_days")
+	private int citationExpiresAfterDays;
+	
+	@Column(name="rights_type")
+	private  String rightsType;
+	
+	@Column(name="rights_value")
+	private String rightsValue;	
+	
+	public Court(){
+		
+	}
+	
+	public Long getId() {
```

Comment:
If we were to use Groovy instead, we could eliminate all these silly getters and setters :/

Reply:
Cool, I'll have to get used to using Groovy.  Fortunately I found that STS will auto generate the getters and setters for me.  I'll look into this.